plugins/com.google.cloud.tools.eclipse.util/META-INF/MANIFEST.MF

```diff
@@ -13,7 +13,6 @@ Export-Package: com.google.cloud.tools.eclipse.util,
  com.google.cloud.tools.eclipse.util.templates.appengine
 Import-Package: com.google.common.base;version="[20.0.0,21.0.0)",
  com.google.common.io;version="[20.0.0,21.0.0)",
- com.google.common.net;version="[20.0.0,21.0.0)",
```

Comment:
Do you get a compile error after applying this PR? Can you find the place where we're importing from here? I didn't see one.